website/source/docs/job-specification/update.html.md

```diff
@@ -37,6 +37,7 @@ job "docs" {
     healthy_deadline  = "5m"
     progress_deadline = "10m"
     auto_revert       = true
+	auto_promote      = true
```

Comment:
The extremely rare whitespace comment I have to make just makes me so thankful for `gofmt`. :sweat_smile: 
```suggestion
    auto_promote      = true
```